extensions/default/src/Panels/WrappedPanelStudyBrowser.jsx

```diff
@@ -20,7 +20,7 @@ function WrappedPanelStudyBrowser({
 }) {
   // TODO: This should be made available a different way; route should have
   // already determined our datasource
-  const dataSource = extensionManager.getDataSources('dicomweb')[0];
+  const dataSource = extensionManager.getDataSources()[0];
```

Comment:
Didn't you add some magic way to get the "default" data source? Or doesn't the mode route publish/identify the selected data source in some way?

Reply:
If you don't give it a tag you get the default. We could add an explicit proxy for the same function also, just saw this explictly assumption and nixxed it as I walked by.

Reply:
So it returns a list of "active data sources", with the first being the default? Providing a name filters, but still gives a list?

A proxy would be neat. If there's enough here for a follow-up issue, great. If not, just resolve this comment thread.

Reply:
It returns the active data source, the datasource might have multiple servers. In our example it returns the list of dicomweb datasources. We have the [0] everywhere throughout the code so we could build with multiple in mind ^^.

Reply:
Ah! Thanks for the refresher.